joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -105,7 +105,9 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
     val signature = Option(s"${typeFullName.getOrElse(Defines.Any)}:($argString)")
 
     val expression = createDotNetNodeInfo(invocationExpr.json(ParserKeys.Expression))
-    val name       = nameFromNode(createDotNetNodeInfo(expression.json(ParserKeys.Name)))
+    val name = expression.node match
```

Comment:
Could we rather make another case within `nameFromNode` instead of add additional custom behaviour here